apb.yml

```diff
@@ -10,7 +10,7 @@ metadata:
   console.openshift.io/iconClass: fa fa-plug
   displayName: Runtime Connector
   dependencies: []
-  documentationUrl: "http://www.aerogear.org/custom-runtime-connector/documentation.html"
+  documentationUrl: "http://www.aerogear.org/external/apb/custom-runtime-connector.html"
```

Comment:
Should this be http://docs.aerogear.org/external/apb/custom-runtime-connector.html ?